.changeset/loud-rules-count.md

```diff
@@ -0,0 +1,6 @@
+---
+"mobx": patch
+---
+
+`trace()`: log when computed becomes suspended
+`requiresObserver` warns instead of throwing
```

Comment:
I didn't see this change in the PR?

Reply:
typo ... `requiresReaction` ... the condition with die is deleted and merged to condition with warn:
```
 warnAboutUntrackedRead_() {
        if (!__DEV__) return
---     if (this.requiresReaction_ === true) {
---             die(`[mobx] Computed value ${this.name_} is read outside a reactive context`)
---     }
        if (this.isTracing_ !== TraceMode.NONE) {
            console.log(
                `[mobx.trace] '${this.name_}' is being read outside a reactive context. Doing a full recompute`
                `[mobx.trace] Computed value '${this.name_}' is being read outside a reactive context. Doing a full recompute.`
            )
        }
---     if (globalState.computedRequiresReaction) {
+++     if (globalState.computedRequiresReaction || this.requiresReaction_) {
            console.warn(
---            `[mobx] Computed value ${this.name_} is being read outside a reactive context. Doing a full recompute`
+++            `[mobx] Computed value '${this.name_}' is being read outside a reactive context. Doing a full recompute.`
            )
        }
    }
```